src/FormattedText.tsx

```diff
@@ -55,7 +55,7 @@ export class FormattedText extends React.Component<IFormattedTextProps, {}> {
             tables: true,
             breaks: false,
             pedantic: false,
-            sanitize: true,
+            sanitize: false,
```

Comment:
Or, something like it. I think it needs to be more flexible. What about an optional hyperlink handler? Let's close issue 199 and open another one for this topic.